File: Helmi Satria/Tugas1.cpp
#include <iostream>

using namespace std;

int main(){
    int count;
    cin >> count;
    int arr[count];
    int counteq = 0;
    // cout << "count : " << count << endl;
    for (int i = 0; i < count; ++i)
    {
    	cin >> arr[i];
    }
    // for (int i = 0; i < count; ++i)
    // {
    // 	cout << "arr[" << i << "] : " << arr[i] << endl;
    // }
    for (int i = 0; i < count; ++i)
    {
    	int kiri = 0;
    	int kanan = 0;
    	for (int j = 0; j < i; ++j)
    	{
    		kiri+= arr[j];
    	}
    	for (int k = i+1; k < count; ++k)
    	{
    		kanan+= arr[k];
    	}
    	if (kiri == kanan){
    		counteq++;
    		cout << i << " ";
    	}
    }
    if (counteq == 0){
    	cout << "-1";
    }
}

File: assignment/assignment_1/1301154674/Tugas1/src/tugas1/Tugas1.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tugas1;

/**
 *
 * @author User
 */
class EquilibriumIndex {
    int equilibrium(int arr[], int n) {
        int i, j;
        int kiri, kanan;
        
        for (i = 0 ; i<n ; i++) {
            kiri = 0;
            kanan = 0;
            
            for (j = 0; j<i; j++) {
                kiri = kiri + arr[j];
            }
            for (j = i+1; j<n; j++) {
                kanan = kanan + arr[j];
            }
            
            if (kiri == kanan) {
                return i;
            }
        }
        return -1;
    }
    public static void main(String[] args) {
        EquilibriumIndex e = new EquilibriumIndex();
        int arr[] = {-7, 1, 5, 2, -4, 3, 0};
        int arr_size = arr.length;
        System.out.println(e.equilibrium(arr, arr_size));
    }
    
}
